### ISSUE TITLE
Document skipPrompts

### ISSUE THREAD

**Issue by degrammer** [stateful/docs.runme.dev#233]
We now support a new frontmatter option: skipPrompts

- [x] Add it to the configuration options here https://docs.runme.dev/configuration#document-options
- [x] Consider adding an InfoBox here explaining that promptEnv can be skipped via this FrontMatter option and add a read more link pointing to https://docs.runme.dev/configuration#cell-options

**Comment by joseeantonior** [comment]
@edeediong This is listed as completed but I don't see it documented here:

- https://docs.runme.dev/configuration/document-level
- https://docs.runme.dev/configuration/cell-level

and only in the /configuration page.

Could you please clarify? Thanks!

**Comment by edeediong** [comment]
@joseeantonior  

Hi Jose! During the release of Shebang, we decided to break down the configurations page as it was overwhelming for any user. If you look at this issue, which you have reopened, you will see that the links Bencho opened are dead. The reason for this ticket was to add a reference/document that the interactive options can be skipped (which the `promptEnv` enables by default). We can’t have a full section on `skipPrompts` without talking about `promptEnv`. When this ticket was handled and closed, everything was on a single page, and there was no need to add extra information about any of the configurations. With the changes to our documentation (plus future documentation), it’s important to understand when tickets are marked as closed. I only moved this ticket recently because you mentioned that the tickets were opened, and if the work was completed, I should move it.  Suppose we need to revisit how skipPrompt is documented. In that case, this is a different conversation, and I’d like clarity on what it is you expect to compare with my notes and see if it is a duplication of efforts, considering the docs page might be changed completely as part of docs 2.0. 

I’ll advise creating a new issue so that we can adequately track work when they’re done and not pick up old stuff and bring them back when there are changes to the docs. It messes up with timelines.
